frontend/src/types/forum-post.tsx

```diff
@@ -0,0 +1,96 @@
+export type ForumPost = {
+    id: number,
+    username: string,
+    title: string,
+    content: string,
+    upvote: number,
+    tags: string[],
+    createdAt: string,
```

Comment:
I think these should be timestamps, and we should render the date based on the unix timestamp. For that, we can use dayjs library.

Reply:
Do you mean they should be of type Date or typs string that are formatted in Unix timestamp format?

Reply:
I think they should be type `number`, and we should get it as number from backend.